neptune_sacred/impl/utils.py

```diff
@@ -0,0 +1,12 @@
+from flatten_dict.flatten_dict import flatten
+
+
+def flatten_space_reducer(k1, k2):
+    if k1 is None:
+        return k2
+    else:
+        return f'{k1} {k2}'
```

Comment:
We join keys by spaces? Spaces in a path? Is this a good practice?

Reply:
I'll ask Prince if there was any particular reason for that.

Reply:
Okay

Reply:
Ok, it's becaue of sacred meta_info keys format, take a look here: https://app.neptune.ai/prince.canuma/sacred-integration/e/SAC-59/all?path=experiment%2Fsacred_config%2Fmeta_info&attribute=options%20--beat-interval

But you're right that it may be confusing in other cases.

We may use `_` except situation when second key starts with `--`, what do you say?

Reply:
Okay, spaces look legit here - I think it's fine the way it is